doc/sphinx/language/core/definitions.rst

```diff
@@ -200,7 +217,12 @@ tactics (see :ref:`writing-proofs`). The user may also enter
 commands to manage the proof mode (see :ref:`proofhandling`).
 
 When the proof is complete, use the :cmd:`Qed` command so the kernel verifies
-the proof and adds it to the global environment.
+the proof and adds it to the global environment. By default, proofs
+that end with :cmd:`Qed` are opaque, that is that their content cannot
```

Comment:
```suggestion
that end with :cmd:`Qed` are :term:`opaque`, that is that their content cannot
```

BTW, I'm preparing a PR that will distinguish in the doc between permanently opaque (Qed) and changeably opaque.  It will be attached to the Opaque command and "opaque" will be defined there.